Source/AviationWeather.Test/Source/framework.h

```diff
@@ -46,6 +46,11 @@ template<> inline std::wstring ToString<uint16_t>(const uint16_t& t) { RETURN_WI
 template<> inline std::wstring ToString<uint16_t>(const uint16_t* t) { RETURN_WIDE_STRING(t); }
 template<> inline std::wstring ToString<uint16_t>(uint16_t* t)       { RETURN_WIDE_STRING(t); }
 
+// aw::metar::visibility
+template<> inline std::wstring ToString<aw::metar::visibility>(const aw::metar::visibility& t) { /* ugh.... */ RETURN_WIDE_STRING(1); }
```

Comment:
These are template specializations for the unit test framework ToString function. Things like Assert::AreEqual use the ToString function to test equality. I have a fix for the reference specialization.